x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/agent_list_page/index.tsx

```diff
@@ -162,30 +281,48 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
         return null;
       },
     },
+    {
+      field: 'active',
+      name: i18n.translate('xpack.ingestManager.agentList.statusColumnTitle', {
+        defaultMessage: 'Status',
+      }),
+      truncateText: true,
+      render: (active: boolean, agent: any) => <AgentHealth agent={agent} />,
+    },
     {
       field: 'config_id',
       name: i18n.translate('xpack.ingestManager.agentList.configColumnTitle', {
-        defaultMessage: 'Agent Config',
+        defaultMessage: 'Configuration',
       }),
       truncateText: true,
       render: (configId: string) => {
         const configName = agentConfigs.find(p => p.id === configId)?.name;
         return configName ? (
           <ConnectedLink color="primary" path={`/configs/${configId}`}>
```

Comment:
Use of `const` for path here as well.

Also,
I'm wondering if the `render()` output for the configuration name used in Agent Config. list should be made a Reusable component and then used here so that both share the same behaviour. In Agent Config, the following was done:

1. Only the name is clickable (not the version)
2. Name is truncated; version is always visible
3. Version is `size=xs`

Reply:
I am going to make a reusable components 👍